src/main/java/io/fabric8/jenkins/openshiftsync/ConfigMapWatcher.java

```diff
@@ -190,6 +196,46 @@ private boolean containsSlave(ConfigMap configMap) {
                 podTemplate = xStream2.fromXML(entry.getValue());
 
                 if( podTemplate instanceof PodTemplate ) {
+                    PodTemplate pt = (PodTemplate)podTemplate;
+                    
+                    // if the image field is and imagestream related ref, without the internal docker registry, prepend it
+                    String image = pt.getImage();
+                    try {
+                        if (!OpenShiftUtils.hasRegistry(image)) {
```

Comment:
i'd definitely prefer some sort of explicit mechanism, whether that is part of the xml itself, an annotation on the configmap, or maybe a prefix in the image value?  `imagestreamtag:myproject/myimagestream:mytag` would be treated as an IST, and an unprefixed value would be treated as an image, perhaps?